### PR TITLE
Rewrite activity to contributions svelte

### PR DESCRIPTION
**Pull request by jum-s**
rename activity -> contributions, to avoid mixing with activity streams

depends of server PR [539](https://github.com/inventaire/inventaire/pull/539)

### CODE REVIEW

File: app/modules/entities/components/contributions.svelte [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<script>
+  import ContributionsPeriod from './contributions_period.svelte'
+</script>
+<div id="activityLayout">
```
Comment (maxlath) [inline app/modules/entities/components/contributions.svelte:4]: Could we keep as a convention to add one empty line as padding between script/template/style blocks? That improves readability imo

File: app/modules/entities/components/contributions.svelte [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <ContributionsPeriod title='last day' period=1/>
+  <ContributionsPeriod title='last week' period=7/>
+  <ContributionsPeriod title='global'/>
+</div>
+<style lang="scss">
+  @import 'app/modules/general/scss/utils';
```
Comment (maxlath) [inline app/modules/entities/components/contributions.svelte:10]:
```suggestion

```
no scss or utils are used here

File: app/modules/entities/components/contributions_period.svelte [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      <div class="cell user {userData.user.deleted ? 'deleted' : ''}">
+        <a href="/u/{userData.user._id}/contributions" class="link" on:click="{e => showUserContributions(e, userData.user)}">{userData.user.username}</a>
+      </div>
+      <div class="histogram">
+        <div class="contributions">
+          {userData.contributions}
+        </div>
+        <div class="bar" style="width: {userData.contributions * 100.0 / highest}px;"></div>
+      </div>
+    </div>
```
Comment (maxlath) [inline app/modules/entities/components/contributions_period.svelte:54]: missing indentation within the each block

File: app/modules/tasks/views/tasks_layout.js [hunk 4]
```diff
@@ -117,10 +117,8 @@ export default Marionette.LayoutView.extend({
         collection: model.suspect.mergeSuggestions,
         currentTaskModel: model
       })
-      if (model.suspect) {
-        this.relativeTasks.show(newRelativeTasks)
-        return this.updateRelativesCount(model)
-      }
+      this.relativeTasks.show(newRelativeTasks)
+      return this.updateRelativesCount(model)
```
Comment (maxlath) [inline app/modules/tasks/views/tasks_layout.js:121]: how is that connected to this PR?

File: app/modules/entities/components/contributions.svelte (hunk 1 above)
Comment (maxlath) [reply app/modules/entities/components/contributions.svelte:4]: Also, `activityLayout` -> `contributonsLayout`?

**Review by maxlath — CHANGES_REQUESTED** [review]
minor details

Also, removing obsolete files such as `app/modules/entities/views/activity_layout.js` could be part of this PR

File: app/modules/entities/components/contributions.svelte (hunk 1 above)
Comment (jum-s) [reply app/modules/entities/components/contributions.svelte:4]: yeah not really, the more it goes the more empty lines only make files longer to scroll, i would actually be for deleting all empty lines of the project :)

Comment (maxlath) [reply app/modules/entities/components/contributions.svelte:4]: many empty lines are the sign that a function is too big and should be split in sub functions, and eventually in different module files (yeah modularization!), but in the case of Svelte components, we can't just split those 3 distinct roles over 3 files, they have to live in the same file, while serving different purposes, so separating by an empty line seems the decent thing to do (and [what people seem to be doing](https://github.com/search?l=Svelte&q=style&type=Code))

Comment (jum-s) [reply app/modules/entities/components/contributions.svelte:4]: those text blocks are distinctive enough to me, from js to html to css, within the first second i can see which is which, therefore an extra line to distinct within the same 'language' makes sense but not here, it only makes you scroll more

**Pull request merged**